src/databricks/labs/ucx/hive_metastore/table_migrate.py

```diff
@@ -458,3 +480,33 @@ def _recreate_view(self, to_view_name, view_text):
         create_sql = f"CREATE VIEW {to_view_name} AS {view_text}"
         logger.info(f"Creating view {to_view_name}")
         self._backend.execute(create_sql)
+
+
+class TableStatusCrawler(CrawlerBase[TableMigrationStatus]):
```

Comment:
```suggestion
class MigrationProgress(CrawlerBase[TableMigrationStatus]):
```
probably a better name

Reply:
OK

Reply:
Done